### ISSUE TITLE
make a stream monitor

### ISSUE THREAD

**Issue by mcfiredrill** [datafruits/fruitbot#14]
Create another worker that polls the stream
`https://streampusher-relay.club/datafruits.mp3`
Every XX minutes or so.
If the request doesn't return 200 , send an alert to discord/chat.

**Comment by einariii** [comment]
right so at least somebody (whoever in the chat) will know when stream is down? what should alert message look like i'll get on it.

**Comment by mcfiredrill** [comment]
Yeah just "stream is down" is fine and if it's sent to a discord channel I can check out why it's down.
